index.bs

```diff
@@ -1744,7 +1744,9 @@ an |ownership type|, a |serialization internal map| and a |realm|:
 
     <dt>|value| has a \[[DateValue]] [=internal slot=].
     <dd>
-      1. Let |serialized| be [=Date.ToDateString=]([=thisTimeValue=](|value|)).
+      1. Set |serialized| to [=Call=]([=Date.toISOString=], |value|).
```

Comment:
As I understand, `toISOString` makes the time UTC regardless of the timezone. Meaning serialization loses part of the information of the original ECMAScript value. Not sure how critical is that though.

Reply:
I don't think that ECMAScript Date values store a timezone; from https://tc39.es/ecma262/#sec-time-values-and-time-range they are just a unix timestamp internally.

Reply:
ok, not an issue.